index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" />
+
+    
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+<header class="header">
+    <div class="header-menu">
+        <nav class="menu-items">
+            <ul>
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+        <div class="hamburger">
+            <i class="fas fa-bars"></i>
+        </div>
+    </div>
+</header>
+
+<div class="headline">
+    <h3>Hey There. I am Alex</h3>
+    <h4>I am a Software Developer</h4>
+    <p class="headline-content">
+        I can help you build a product, feature or website Look through some of my work and
+        <br /> experience! If you like what you see and have a project you need coded,<br>
+        don’t hesitate to contact me.
+    </p>
+    <div class="social-icons">
+        <a href="#"><img src="./assets/icons/github.svg" alt="GitHub icon"></a>
+        <a href="#"><img src="./assets/icons/linkedin.svg" alt="LinkedIn icon"></a>
+        <a href="#"><img src="./assets/icons/angellist.svg" alt="AngelList icon"></a>
+        <a href="#"><img src="./assets/icons/twitter.svg" alt="Twitter icon"></a>
+        <a href="#"><img src="./assets/icons/medium.svg" alt="Medium icon"></a>
```

Comment:
- [ ] I think it would be better also for these list of icons to use a ```ul``` tag to group them, and ```li``` per icon; to explicitly mean a list of icons and also for the best HTML semantics 😎😎 with and ```a``` tag on each 👍 

_Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href._

Reply:
Excelent,I have updated the social icons and used a ul/li tags instead of custom classes in divs.